wd_helper.au3

```diff
@@ -2196,6 +2196,43 @@ Func _WD_CheckContext($sSession, $bReconnect = Default, $vTarget = Default)
 	Return SetError(__WD_Error($sFuncName, ($iResult) ? $_WD_ERROR_Success : $_WD_ERROR_Exception), 0, $iResult)
 EndFunc   ;==>_WD_CheckContext
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_FindElement_ByRegExp
+; Description ...: Find element by matching attributes values using regular expression
+; Syntax ........: _WD_FindElement_ByRegExp($sSession, $sMode, $sRegEx[, $sRegExFlags = ""[, $bAll = False]])
+; Parameters ....: $sSession            - Session ID from _WD_CreateSession
+;                  $sMode               - Attribute of the element which should be matched, e.g. `id`, `style`, `class` etc.
+;                  $sRegEx              - RegEx in JavaScript format
+;                  $sRegExFlags         - [optional] RegEx Flags. Default is "".
+;                  $bAll                - [optional] Fetch a list of all elements that fits to the RegEx pattern ? Default is False - return only first element
+; Return values .: Success - Matched element
+;                  Failure - Response from _WD_ExecuteScript() and sets @error to value returned from _WD_ExecuteScript()
+; Author ........: TheDcoder
+; Modified ......: mLipok
+; Remarks .......: The RegEx matching is done by the browser's JavaScript engine so AutoIt's RegEx rules may not accurately work
+;                  in this function. You may refer to this cheatsheet for further information:
+;                  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Cheatsheet
+;                  The returned matched elements are in the same format like result form _WD_FindElement and _WD_WaitElement
```

Comment:
`like result` or `like results` ?

Reply:
`results`. Also `form` should be `from`. I would propose to change this sentence to something closer to this--

`The matching elements are returned in the same format as the results from _WD_FindElement`

Reply:
O

Reply:
Please take a look:
[description rewording](https://github.com/Danp2/au3WebDriver/pull/301/commits/6974f29d5e2769255c9a52e7eee83b93d8730047)